testdata/docker/VERSION

```diff
@@ -1 +1 @@
-0.1.0
+1614356656
```

Comment:
I can't actually push, docker gives me `unauthorized: access to the requested resource is not authorized`. Can you take another look?

Reply:
Nvm, I had to log in first :see_no_evil: